README.md

```diff
@@ -72,10 +67,8 @@ rustls does not and will not support:
 * Ciphersuites without forward secrecy.
 * Renegotiation.
 * Kerberos.
-* Compression.
 * Discrete-log Diffie-Hellman.
 * Automatic protocol version downgrade.
-* Self-signed certificates.
```

Comment:
I think we should keep these non-features. Folks ask about support for these things otherwise. The self-signed certificate non-support item was added recently for this reason: https://github.com/rustls/rustls/pull/1382

I think any nuance that applies to whether compression or self-signed certificates are supported likely applies to the other non-features like discrete-log DH, and renegotiation, no?

Reply:
Like I said in that other PR, I think people will read "self-signed certificates" here and conclude that rustls cannot be configured to turn off certificate checking. And I think if we start qualifying some of these items the list will become a bit of an exposition which doesn't really fit in a README.

Maybe we could add a section to https://docs.rs/rustls/latest/rustls/manual/_03_howto/index.html which covers the detail.

Reply:
> Maybe we could add a section to https://docs.rs/rustls/latest/rustls/manual/_03_howto/index.html which covers the detail.

That sounds reasonable to me.